website/content/api-docs/secret/pki.mdx

```diff
@@ -3018,14 +3025,20 @@ the CRL.
 - `expiry` `(string: "72h")` - The amount of time the generated CRL should be valid.
 - `disable` `(bool: false)` - Disables or enables CRL building.
 - `ocsp_disable` `(bool: false)` - Disables or enables the OCSP responder in Vault.
+- `auto_rebuild` `(bool: false)` - Enables or disables periodic rebuilding of
+  the CRL upon expiry.
+- `auto_rebuild_grace_period` `(string: "12h")` - Grace period before CRL expiry
```

Comment:
Is 12h an standard within the industry, it seems rather long?

Reply:
I don't know that any of this is standard within any industry. :-)

`12h` is time the time prior to CRL expiry that we attempt rebuilding... It really doesn't matter a whole lot, but with a longer grace period, you're more likely to catch issues with CRL rebuilds and fix it before the CRL goes stale. I'm open to suggestions on default timing, but I figure we should probably make it a larger window than a smaller one by default and let users customize it as they see fit. :-)